portal/src/texts/getstarted/PortalGuide.md

```diff
@@ -6,21 +6,22 @@ order: 6
 
 # Dokumentasjon til portalen
 
-For at et designsystem skal fungere, må dokumentasjonen av systemet holde høy kvalitet. Hvis utviklere og designer ikke vet hvilke muligheter og begrensninger som finnes i systemet, kommer man til å gjøre feil og dobbeltarbeid. Vi i kjerneteamet setter stor stolthet i å levere Jøkul med god dokumentasjon. Vi har bygd dokumentasjonsriggen selv for å sikre oss at den gir oss akkurat det vi trenger for Jøkul og for å gjøre utvikleropplevelsen så smidig som mulig. Det skal bare være én kilde til sannhet.
+For at et designsystem skal fungere, må systemdokumentasjonen være god. Hvis utviklere og designere ikke vet hvilke muligheter og begrensninger som finnes i systemet, kommer de til å gjøre feil og dobbeltarbeid. 
 
-## Komponenteksempler
+Vi i kjerneteamet legger vår stolthet i å levere Jøkul med god dokumentasjon. Vi har bygd opp dokumentasjonen selv, for å sikre at den inneholder akkurat det vi trenger. Slik tror vi at utvikling og design med Jøkul blir en smidig prosess. Det skal bare være én kilde til sannhet.
```

Comment:
Har har det også skjedd en liten endring i mening. Med "dokumentasjonsriggen" menes teknologien vi har brukt for å implementere selve portalen. Det er nok mulig man bør finne et bedre ord, men det er viktig å skille det fra "dokumentasjonen", som refererer til selve innholdet 😊